src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp

```diff
@@ -350,13 +351,21 @@ bool FixConservatives::operator()(
                                          : static_cast<SimdType>(0);
 
     // Equation B.27 of Foucart
+    //
+    // We avoid division by zero in the SIMD case using a mask and effectively
+    // set the result to zero by dividing by the max double.
```

Comment:
It looks like you decided on a different implementation than this, without the max double part.